spirv_glsl.hpp

```diff
@@ -91,6 +91,8 @@ class CompilerGLSL : public Compiler
 		{
 			// In vertex shaders, rewrite [0, w] depth (Vulkan/D3D style) to [-w, w] depth (GL style).
 			bool fixup_clipspace = true;
+			// Inverts the Y coordinate so that the [-1, -1] pixel corresponds to texture coordinate [0, 0]
+			bool flip_y = false;
```

Comment:
Probably best to call this flip_vert_y to match HLSL?
Also, the comment seems a bit weird. In GL, [-1, -1] already maps to window space [0, 0], and hence texture coordinate [0, 0].